### PR TITLE
tests(felt): add tests for num_trait::Zero trait

### PR DESCRIPTION
**Pull request by oboulant**
# add tests for num_trait::Zero traits

This PR takes a bit of https://github.com/lambdaclass/cairo-rs/issues/823 in adding tests for the num_trait::Zero trait implementation for Felts : 
* `zero()`
* `is_zero()`

It only affects the `felt` library.

## Checklist
- [x] Linked to Github Issue
- [x] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
- [ ] Documentation has been added/updated.

### CODE REVIEW

File: felt/src/lib.rs [hunk 1]
```diff
@@ -1059,4 +1059,19 @@ mod test {
...
+        let felt_zero = Felt::zero().to_biguint();
+        assert_eq!(zero, felt_zero)
+    }
+
+    #[test]
+    fn is_zero() {
+        let felt_zero = Felt::zero();
+        let felt_non_zero = Felt::new(3);
+        assert_eq!(true, felt_zero.is_zero());
+        assert_eq!(false, felt_non_zero.is_zero())
```
Comment (Juan-M-V) [inline felt/src/lib.rs:1075]:
```suggestion
        assert!(felt_zero.is_zero());
        assert!(!felt_non_zero.is_zero())
```

**Review by Jrigada — APPROVED** [review]
